cmd/create/execution_util.go

```diff
@@ -93,7 +93,23 @@ func createExecutionRequestForTask(ctx context.Context, taskName string, project
 
 func relaunchExecution(ctx context.Context, executionName string, project string, domain string,
 	cmdCtx cmdCore.CommandContext) error {
-	relaunchedExec, err := cmdCtx.AdminClient().RelaunchExecution(ctx, &admin.ExecutionRelaunchRequest{
+	recoveredExec, err := cmdCtx.AdminClient().RelaunchExecution(ctx, &admin.ExecutionRelaunchRequest{
+		Id: &core.WorkflowExecutionIdentifier{
+			Name:    executionName,
+			Project: project,
+			Domain:  domain,
+		},
+	})
+	if err != nil {
+		return err
+	}
+	fmt.Printf("execution identifier %v\n", recoveredExec.Id)
+	return nil
+}
+
+func recoverExecution(ctx context.Context, executionName string, project string, domain string,
+	cmdCtx cmdCore.CommandContext) error {
+	relaunchedExec, err := cmdCtx.AdminClient().RecoverExecution(ctx, &admin.ExecutionRecoverRequest{
```

Comment:
Shouldn't this be recoveredExec and L96 be relaunchedExec

Reply:
hah yes, thanks! done